### ISSUE TITLE
Analyze Market Price

### ISSUE THREAD

**Issue by claytonliotti** [binary-com/binary-bot#1891]
Hello
It will be possible to make an analysis of the market price.
high and low example through ticks

sorry about my English

**Comment by binary-ex-machina** [comment]
Yes it's possible. You need to select a period to get the highest and lowest

![image](https://user-images.githubusercontent.com/34610787/64183393-368dc400-ce40-11e9-8da6-0f3d1d1d068a.png)

**Comment by claytonliotti** [comment]
ok thanks, I will test
